tests/integration/duplicity/duplicity_spec.rb

```diff
@@ -34,7 +37,46 @@
     its('content') { should match /PASSPHRASE='your-super-secret-passphrase'/ }
     its('content') { should match /AWS_ACCESS_KEY_ID='your-access-key-id'/ }
     its('content') { should match /AWS_SECRET_ACCESS_KEY='your-secret-access-key'/ }
-    its('content') { should match /ftp:\/\/user:pass@your-server.com\/mybackup/ }
```

Comment:
I think it might be wise to use two tests, one with a simple and one with the multi configurationn, so we can be sure that changes to multi do not mess with the simple backend configs. I suppose we'd need two test scenarios then, though